docs/docs/porting-create-react-app-to-gatsby.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Porting Create React App to Gatsby
+---
+
+## What is Create React App?
+
+Create React App is an officially supported tool from React to setup React apps, without having to deal with complicated configurations. It provides a default setup for tools like webpack and Babel that are useful in a modern development pipeline.
+
+Gatsby is similar in that it can also help you setup an app and removes much of the configuration headache, but offers some additional advantages like the ability to leverage Gatsby's ecosystem and performance optimizations. React's own [docs](https://reactjs.org/) run on Gatsby, and React even [recommends Gatsby](https://github.com/facebook/create-react-app#popular-alternatives) to users of Create React App!
+
+---
+
+## What do you get by changing to Gatsby?
+
+Both Create React App and Gatsby use React and allow a faster setup, especially in setting up the initial configuration of a site, but their are differences.
+
+### Performance optimizations
+
+A lot of automatic optimization upgrades like automatic code splitting and preloading of data come out of the box with Gatsby to improve site speed and performance audit scores. There are improvements like progressive web app support that are part of both projects, but Gatsby offers even more specific performance optimizations.
+
+A few examples:
+
+- `gatsby-link` uses an [Intersection Observer to preload linked pages](/blog/2019-04-02-behind-the-scenes-what-makes-gatsby-great/#intersectionobserver) when they appear in the viewport, making them feel like they load _instantly_
+- `gatsby-image` will create optimized versions of your images in different sizes, loading a lighter version of an image and replacing it with a higher res version when loading has finished, check out [the demo](https://using-gatsby-image.gatsbyjs.org/) to see for yourself
+
+These and more work together to make your Gatsby site fast by default.
+
+### Expansive ecosystem
+
+Gatsby still works with all the `react` packages running in a Create React App project, but adds the ability to leverage [hundreds of other Gatsby plugins](/plugins) to drop in additional features for things like adding SEO, responsive images, RSS feeds, or much more.
+
+### Unified GraphQL data layer
+
+Plugins can also pull in data from any number of sources like APIs, CMSes, or the filesysten. That data is combined into a unified data layer that you can query with GraphQL throughout your app.
+
+This data layer simplifies the process of pulling data from different sources and providing them in your pages and components.
```

Comment:
Perhaps we could introduce the Content Mesh here? https://www.gatsbyjs.org/blog/2018-10-04-journey-to-the-content-mesh/

Reply:
I like that idea, will do!